docs/usage/configuration-options.md

```diff
@@ -823,7 +823,7 @@ Also, approval rules overriding should not be [prevented in GitLab settings](htt
 
 ## golang
 
-Configuration added here applies for all Go-related updates, however currently the only supported package manager for Go is the native Go Modules (the `gomod` manager).
+Configuration added here applies for all Go-related updates, but currently the only supported package manager for Go is the native Go Modules (the `gomod` manager).
```

Comment:
```suggestion
Configuration added here applies for all Go-related updates.
The only supported package manager for Go is the native Go Modules (the `gomod` manager).
```

Let's rewrite so we don't need `but/however` here. 😄 

I think our readers expect our docs to mention things that are _currently_ supported, so we can drop `currently` as well. 😉